### PR TITLE
examples: typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Fix a few typos in code comment in examples.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ljflores** [comment]
jenkins retest this please

File: examples/boto3/list_unordered.py [hunk 1]
```diff
@@ -20,6 +20,6 @@
         aws_access_key_id=access_key,
         aws_secret_access_key=secret_key)
 
-# geting an unordered list of objets is an extension to AWS S3 API
+# getitng an unordered list of objects is an extension to AWS S3 API
```
Comment (ljflores) [inline examples/boto3/list_unordered.py:23]: Did you mean to write "getting" here?

Comment (DimitriPapadopoulos) [reply examples/boto3/list_unordered.py]: Ooops... Sorry about that. Fixed now.

**Review by ljflores — APPROVED** [review]
@neha-ojha if this passes the tests, it can be merged

**Comment by neha-ojha** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
jenkins test make check

**Pull request merged**